src/DebugSceneList.js

```diff
@@ -120,12 +122,18 @@ class DebugSceneList extends Component {
   }
 
   renderSceneModal() {
+    let component = this.state.selectedItem.component;
+    const closePropName = this.state.selectedItem.closePropName;
+    console.log('closePropName', closePropName);
+    if (closePropName) {
+      component = React.cloneElement(component, {[closePropName]: this.onHideScene});
+    }
     return (
```

Comment:
I like it! Would be even nicer with a short comment explaining what's going on here